apollo-router/src/executable.rs

```diff
@@ -435,11 +438,10 @@ impl Executable {
             },
         };
 
-        let is_telemetry_disabled = std::env::var("APOLLO_TELEMETRY_DISABLED").ok().is_some();
-        let apollo_telemetry_msg = if is_telemetry_disabled {
-            "Anonymous usage data was disabled via APOLLO_TELEMETRY_DISABLED=1.".to_string()
+        let apollo_telemetry_msg = if opt.anonymous_telemetry_disabled {
+            "anonymous usage data was disabled.".to_string()
         } else {
-            "Anonymous usage data is gathered to inform Apollo product development.  See https://go.apollo.dev/o/privacy for more info.".to_string()
```

Comment:
It's because Rust log style is lower case. It should also be missing the final full stop, so I'll fix that.

Reply:
I want to strongly push back on this from our user experience perspective.  Rust's stylistic preferences do request this in some cases (though my understanding is that this is for panic error message content on the premise that it will be prefixed with a `Capitalized prefix:`), but the Router's log output is not merely for Rust-aware users nor is log output in products built with Rust needing to subscribe to this.  It still comes down to user experience, and a CLI written in Rust should use its style guidelines.

I think this information log output and needs to be well formatted, properly punctuated and capitalized.  Frankly, in my opinion a sentence at the start of a line in a console output that is lower cased looks unprofessional.

We should solve this outside of this PR, but this change is included in this PR so let's just undo it for now and discuss?